amy/mta/mta.rb

```diff
@@ -0,0 +1,161 @@
+#creates a subway_lines hash
+
+subway_lines = {
+
+	"N" => ["Times Square", "34th", "28th(N)", "23rd(N)", "Union Square", "8th(N)"],
+	"L" => ["8th(L)", "6th", "Union Square", "3rd", "1st"],
+	"6" => ["Grand Central", "33rd", "28th(6)", "23rd(6)", "Union Square", "Astor Place"]
+}
+
+#gets user inputs and stores them in variables
+
+puts "Which line are you getting on at? "
+puts subway_lines.keys
+print '> '
+line_on = gets.chomp.upcase
+
+puts "Which stop are you getting on at?"
+if line_on == "N"
+	puts subway_lines["N"]
+elsif line_on == "L"
+	puts subway_lines["L"]
+elsif line_on == "6"
+	puts subway_lines["6"]
+else
+	puts "No such line!"
+end
+print '> '
```

Comment:
instead of using a if statement here you can just puts the line with user input as the key. 

    puts subway_lines[line_on]